src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -299,4 +357,40 @@ public static Translation2d[] getModuleTranslations() {
       new Translation2d(-TRACK_WIDTH_X / 2.0, -TRACK_WIDTH_Y / 2.0)
     };
   }
+
+  public Rotation2d getRotationToTranslation(Translation2d translation) {
+
+    double angle = Math.atan2(translation.getY() - pose.getY(), translation.getX() - pose.getX());
+    return Rotation2d.fromRadians(angle);
+  }
+
+  public Command pointTowardsTranslation(DoubleSupplier x, DoubleSupplier y) {
+    ProfiledPIDController headingController =
```

Comment:
make sure having this as a member doesnt break it if you have multiple pointTowardsTranslation calls or call the method twice